htdocs/API/2.0/index.php

```diff
@@ -10,6 +10,21 @@
 
 require_once 'configuratie.include.php';
 
+/**
+ * Maak de API toegankelijk vanaf bepaalde externe domeinen.
+ */
+if (in_array($_SERVER['HTTP_ORIGIN'], explode(',', API_ORIGINS), true)) {
+	header('Access-Control-Allow-Origin: ' . $_SERVER['HTTP_ORIGIN']);
+	header('Access-Control-Max-Age: 1440');
+	header('Access-Control-Allow-Headers: Accept, Origin, Content-Type, X-Csr-Authorization');
+	header('Access-Control-Allow-Methods: PUT, GET, POST, DELETE, OPTIONS');
+
+	if ($_SERVER['REQUEST_METHOD'] === 'OPTIONS') {
+		http_response_code(204);
+		exit;
+	}
+}
```

Comment:
Ander punt, als `Access-Control-Allow-Origin` kan veranderen, stuur dan voor de volledigheid ook een `Vary: Origin` header.

Citaat van MDN:

## CORS and caching

>If the server specifies an origin host rather than "*", then it must also include Origin in the Vary response header to indicate to clients that server responses will differ based on the value of the Origin request header.

```
Access-Control-Allow-Origin: https://developer.mozilla.org
Vary: Origin
```
@Manduro